client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx

```diff
@@ -11,13 +11,16 @@ import styles from './BatchChangesListIntro.module.scss'
 export const BatchChangesChangelogAlert: React.FunctionComponent = () => (
     <DismissibleAlert
         className={styles.batchChangesListIntroAlert}
-        partialStorageKey="batch-changes-list-intro-changelog-3.37"
+        partialStorageKey="batch-changes-list-intro-changelog-3.39"
     >
         <Card className={classNames(styles.batchChangesListIntroCard, 'h-100')}>
             <CardBody>
-                <h4>Batch Changes updates in version 3.37</h4>
+                <h4>Batch Changes updates in version 3.39</h4>
```

Comment:
This is failing accessibility audit as it's now included in the integration test and being used in a place where there is no H2 or H3 on the page, so including a H4 breaks the heading order.

We can keep the styles but just change the element using the WIldcard `H4/H2` elements:

```suggestion
                <H4 as={H2}>Batch Changes updates in version 3.39</H4>
```

Reply:
oh lord, the log output is so strange 🤦 It says duplicate run further up 🙃 I'll get it fixed, thanks for taking a look @umpox!